include/odp/api/spec/event.h

```diff
@@ -209,6 +209,41 @@ void odp_event_free_multi(const odp_event_t event[], int num);
  */
 void odp_event_free_sp(const odp_event_t event[], int num);
 
+/**
+ * Event flow id value
+ *
+ * Returns the flow id value set in the event.
+ * Flow id value set in the event is used by scheduler to distribute the
+ * event across different flows. The returned id is either the platform
+ * generated (if any), or if odp_event_flow_id_set() were called then the value
+ * set there.
+ *
+ * @param	event	Event handle
+ *
+ * @return		Hash value
+ *
+ * @note The initial event flow id generated by the HW will be same as flow
+ * hash generated for packets. The hash algorithm and the header fields
+ * defining the flow (therefore used for hashing) is platform dependent.
```

Comment:
This note text can be moved above (to the specification text proper). Initial flow id is zero when not set. For packets received from packet input, initial flow id value is determined by the flow hash configuration...

So, it's only for input packets that flow id is set by pktin hash (if configured to do so).

Reply:
@psavol I've changed the description, could you please take a look.